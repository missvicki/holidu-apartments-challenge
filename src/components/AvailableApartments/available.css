.available__container {
    display: flex;
    justify-content: space-between;
    margin-top: 24px;
    background: teal;
    padding: 15px;
}
  
.left__results {
    flex: 2;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    font-family: "Roboto";
}
.available-results{
    color: white;
}

.results__title {
    font-size: 2rem;
    font-weight: bold;
}
