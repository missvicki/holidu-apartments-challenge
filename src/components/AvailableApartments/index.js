import React from 'react';
import './available.css';

const AvailableApartments = ({ totalResults }) => (
  <div className="available__container">
    <div className="left__results">
      <div className="results__title">Available Apartments</div>
      <div className="available-results">{`Showing ${totalResults} results`}</div>
    </div>
  </div>
);

export default AvailableApartments;