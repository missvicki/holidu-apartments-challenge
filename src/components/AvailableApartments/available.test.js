import React from 'react';
import { shallow } from 'enzyme';
import AvailableApartments from './index';

describe('Available', () => {
  it('should not have regressed', () => {
    const component = shallow(<AvailableApartments />);

    expect(component).toMatchSnapshot();
  });
});
