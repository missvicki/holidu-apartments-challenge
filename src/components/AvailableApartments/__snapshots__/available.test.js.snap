// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Available should not have regressed 1`] = `""`;
